tools/depends/target/mariadb/Makefile

```diff
@@ -16,8 +24,8 @@ $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	rm -rf $(PLATFORM); mkdir -p $(PLATFORM)/build
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
 	cd $(PLATFORM); patch -p1 -i ../01-android.patch
-	sed -ie 's| "DYNAMIC" | "STATIC" |' $(PLATFORM)/cmake/plugins.cmake
-	cd $(PLATFORM)/build; $(CMAKE) -DAUTH_GSSAPI=OFF -DWITH_UNITTEST:BOOL=OFF -DWITH_EXTERNAL_ZLIB:BOOL=ON -DWITH_CURL:BOOL=OFF ..
+	cd $(PLATFORM); patch -p1 -i ../02-symbol_rename-gnutls_clash.patch
+	cd $(PLATFORM)/build; $(CMAKE) -DWITH_SSL=ON ${CONFIGURE_FLAGS} -DWITH_UNIT_TESTS:BOOL=OFF -DWITH_EXTERNAL_ZLIB:BOOL=ON -DWITH_CURL:BOOL=OFF ..
```

Comment:
Why not name the value of CONFIGURE_FLAGS variable explicitly here?

Reply:
Hope this is what you were thinking.